enclave-runtime/src/attestation.rs

```diff
@@ -121,6 +133,46 @@ pub unsafe extern "C" fn perform_ra(
 	}
 }
 
+#[no_mangle]
+pub unsafe extern "C" fn mock_register_enclave_xt(
+	w_url: *const u8,
+	w_url_size: u32,
+	unchecked_extrinsic: *mut u8,
+	unchecked_extrinsic_size: u32,
+) -> sgx_status_t {
```

Comment:
It's nice to have this function moved here now 👍 I think it makes more sense to have them grouped together.